google-cloud-datastore/lib/google/cloud/datastore/dataset/aggregate_query_results.rb

```diff
@@ -0,0 +1,115 @@
+
+module Google
+  module Cloud
+    module Datastore
+      class Dataset
+        ##
+        # # AggregateQueryResults
+        #
+        # An AggregateQueryResult object is a representation for
+        # a result of an AggregateQuery or a GqlQuery.
+        #
+        # @example
+        #   require "google/cloud/datastore"
+        #
+        #   datastore = Google::Cloud::Datastore.new
+        #
+        #   query = Google::Cloud::Datastore::Query.new
+        #   query.kind("Task")
+        #        .where("done", "=", false)
+        #
+        #   Create an aggregate query
+        #   aggregate_query = query.aggregate_query
+        #                          .add_count
+        #
+        #   aggregate_query_results = dataset.run_aggregation aggregate_query
+        #   puts aggregate_query_results.get
+        #
+        class AggregateQueryResults
+          ##
+          # Read timestamp the query was done on the database at.
+          #
+          # @return Google::Protobuf::Timestamp
+          attr_reader :read_time
+
+          ##
+          # Retrieves the aggregate data.
+          #
+          # @param aggregate_alias [String] The alias used to access
+          #   the aggregate value. For an AggregateQuery with a
+          #   single aggregate field, this parameter can be omitted.
+          #
+          # @return [Integer] The aggregate value.
+          #
+          # @example
+          #   require "google/cloud/datastore"
+          #
+          #   datastore = Google::Cloud::Datastore.new
+          #
+          #   query = Google::Cloud::Datastore::Query.new
+          #   query.kind("Task")
+          #        .where("done", "=", false)
+          #
+          #   Create an aggregate query
+          #   aggregate_query = query.aggregate_query
+          #                          .add_count
+          #
+          #   aggregate_query_results = dataset.run_aggregation aggregate_query
+          #   puts aggregate_query_results.get
+          #
+          # @example Alias an aggregate query
+          #   require "google/cloud/datastore"
+          #
+          #   datastore = Google::Cloud::Datastore.new
+          #
+          #   query = Google::Cloud::Datastore::Query.new
+          #   query.kind("Task")
+          #        .where("done", "=", false)
+          #
+          #   Create an aggregate query
+          #   aggregate_query = query.aggregate_query
+          #                          .add_count aggregate_alias: 'total'
+          #
+          #   aggregate_query_results = dataset.run_aggregation aggregate_query
+          #   puts aggregate_query_results.get('total')
+          def get aggregate_alias = nil
+            if @aggregate_fields.count > 1 && aggregate_alias.nil?
+              raise ArgumentError, "Required param aggregate_alias for AggregateQuery with multiple aggregate fields"
+            end
+            aggregate_alias = @aggregate_fields.keys.first if aggregate_alias.nil?
```

Comment:
Just as a Ruby idiom nit: the following is generally more idiomatic:

```ruby
aggregate_alias ||= @aggregate_fields.keys.first
```

Reply:
Thanks! Changed it to the idiomatic way.